src/mono/wasi/Wasi.Build.Tests/WasiTemplateTests.cs

```diff
@@ -138,6 +138,47 @@ public void ConsolePublishAndRunForSingleFileBundle(string config, bool relinkin
 
     }
 
+    [Theory]
+    [InlineData("Debug", /*appendRID*/ true, /*useArtifacts*/ false)]
+    [InlineData("Debug", /*appendRID*/ true, /*useArtifacts*/ true)]
+    [InlineData("Debug", /*appendRID*/ false, /*useArtifacts*/ false)]
+    [InlineData("Debug", /*appendRID*/ false, /*useArtifacts*/ true)]
+    public void ConsoleBuildAndRunForDifferentOutputPaths(string config, bool appendRID, bool useArtifacts)
+    {
+        string extraPropertiesForDBP = "";
+        if (appendRID)
+            extraPropertiesForDBP += "<AppendRuntimeIdentifierToOutputPath>true</AppendRuntimeIdentifierToOutputPath>";
+        if (useArtifacts)
+            extraPropertiesForDBP += "<UseArtifactsOutput>true</UseArtifactsOutput><ArtifactsPath>.</ArtifactsPath>";
```

Comment:
Are these two always required together? r what is the usage? What happens if we pass `<ArtifactsPath>.</ArtifactsPath>` but leave `UseArtifactsOutput` as false? If nothing, then why do we need both values?

Reply:
Yep, both are needed. If you only set `UseArtifactsOutput` then it complains about `ArtifactsPath` being unset. The other way around probably does nothing at all.